### PR TITLE
Remove pause button outline when button is clicked

### PR DESCRIPTION
**Pull request by JillianK**
Use the [focus-visible](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible) pseudoclass to hide the outline when the pause/play button is clicked while leaving the outline behavior unchanged for keyboard navigation.

In edit mode with keyboard navigation:
Screen Shot 2021-04-02 at 1 34 30 PM

In edit mode with click:
Screen Shot 2021-04-02 at 1 33 48 PM

In view mode with keyboard navigation:
Screen Shot 2021-04-02 at 1 35 11 PM

In view mode with click:
Screen Shot 2021-04-02 at 1 35 23 PM

One thing to note is that this isn't supported in every browser yet. [Here](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible#browser_compatibility) is the browser compatibility list.

## Links

- jira ticket: [STAR-1510](https://codedotorg.atlassian.net/browse/STAR-1510?atlOrigin=eyJpIjoiMjE3NGYxZjczY2Y3NGM1ZWE5YjhjNWY4ZWZkYzk0NGYiLCJwIjoiaiJ9)
- :focus-visible docs: [here](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/style/common.scss [hunk 1]
```diff
@@ -107,6 +107,10 @@ html[dir='rtl'] #visualizationColumn {
   }
 }
 
+#pause-button:focus:not(:focus-visible) {
```
Comment (epeach) [inline apps/style/common.scss]: Open question here - if we want to reuse this focus-visibility in other settings on other buttons, I'm wondering if we should add this to something more generic?

Comment (JillianK) [reply apps/style/common.scss]: I can make a class for it instead of using an ID

File: apps/style/common.scss [hunk 2]
```diff
@@ -107,7 +107,7 @@ html[dir='rtl'] #visualizationColumn {
   }
 }
 
-#pause-button:focus:not(:focus-visible) {
+.accessibility-friendly-no-focus-outline:focus:not(:focus-visible) {
```
Comment (epeach) [inline apps/style/common.scss]: Nit: Let's make this class name a little shorter to be more eng-friendly and then let's add a comment here to document the rest of the context.

**Review by epeach — APPROVED** [review]
Awesome - this looks great!

**Pull request merged**